cmd/hamctl/command/rollback.go

```diff
@@ -62,7 +62,7 @@ has no effect.`,
 
 				searcher := func(input string, index int) bool {
 					release := items[index]
-					name := strings.ToLower(fmt.Sprintf("#%v %s", release.ReleaseIndex, release.ArtifactID))
+					name := strings.ToLower(fmt.Sprintf("#%v %s", release.ReleaseIndex+1, release.ArtifactID))
```

Comment:
I think this needs explanation. Why the +1?

Reply:
It's to fix the UI: 
![image](https://user-images.githubusercontent.com/56017845/179726313-25302ebc-1cd3-4392-9d15-7df2b72fca68.png)

Reply:
I think we should add a comment in the code to explain this, eg. Using index 1 to make a more semantic listing to users.

Reply:
Would we also need to change the[ input index](https://github.com/lunarway/release-manager/blob/master/cmd/hamctl/command/rollback.go#L80), right?